rosetta-source/src/main/resources/cdm-sample-files/functions/sec-lending/reallocation/reallocation-pre-settled-func-input.json

```diff
@@ -1,759 +1,665 @@
+            } ],
+            "direction" : "Replace"
+          }
+        }, {
+          "quantityChange" : {
+            "change" : [ {
+              "quantity" : [ {
+                "value" : {
+                  "value" : 1500000.000,
+                  "unit" : {
+                    "currency" : {
+                      "value" : "USD"
                     }
                   }
-                ],
-                "direction": "Replace"
-              }
-            }
-          ]
-        }
-      },
-      "before": {
-        "value": {
-          "trade": {
-            "tradeIdentifier": [
-              {
-                "issuer": {
-                  "value": "KTKL"
                 },
-                "assignedIdentifier": [
-                  {
-                    "identifier": {
-                      "value": "0266001-allocation-lender-2"
-                    }
+                "meta" : {
+                  "location" : [ {
+                    "scope" : "DOCUMENT"
+                  } ]
+                }
+              }, {
+                "value" : {
+                  "value" : 60000.000,
+                  "unit" : {
+                    "financialUnit" : "Share"
                   }
-                ],
-                "meta": {
-                  "globalKey": "c76d61e6"
+                },
+                "meta" : {
+                  "location" : [ {
+                    "scope" : "DOCUMENT"
+                  } ]
                 }
+              } ],
+              "meta" : {
+                "globalKey" : "b1506fcf"
               }
-            ],
-            "tradeDate": {
-              "value": "2020-09-21",
-              "meta": {
-                "globalKey": "3f2255"
-              }
+            } ],
+            "direction" : "Replace"
+          }
+        } ]
+      }
+    },
+    "before" : {
+      "value" : {
+        "trade" : {
+          "tradeIdentifier" : [ {
+            "issuer" : {
+              "value" : "KTKL"
             },
-            "tradableProduct": {
-              "product": {
-                "contractualProduct": {
-                  "productTaxonomy": [
-                    {
-                      "source": "ISDA",
-                      "productQualifier": "SecurityLendingAgreement"
+            "assignedIdentifier" : [ {
+              "identifier" : {
+                "value" : "0266001-allocation-lender-2"
+              }
+            } ],
+            "meta" : {
+              "globalKey" : "c76d61e6"
+            }
+          } ],
+          "tradeDate" : {
+            "value" : "2020-09-21",
+            "meta" : {
+              "globalKey" : "3f2255"
+            }
+          },
+          "tradableProduct" : {
+            "product" : {
+              "contractualProduct" : {
+                "productTaxonomy" : [ {
+                  "source" : "ISDA",
+                  "productQualifier" : "SecurityLendingAgreement"
```

Comment:
This one still says "SecurityLendingAgreement"? (There may be other ones, I haven't checked them all)

Reply:
Three files updated